crates/symbolicator-service/src/services/symbolication/source_context.rs

```diff
@@ -0,0 +1,98 @@
+pub const DEFAULT_CONTEXT_LINES: usize = 5;
+
+/// Resolves the source context around `lineno` from the `source` file.
+///
+/// N (`context_lines`) lines of context will be resolved in addition to `lineno`.
+/// This defaults to [`DEFAULT_CONTEXT_LINES`].
+/// If `trim_to_column` is provided, every output line will be truncated to ~150 characters
+/// centered around the provided column number, and a `{snip}` marker is added at the trimmed edges.
+///
+/// If no source line for `lineno` is found in `source`, it will return [`None`] instead.
+pub fn get_context_lines(
+    source: &str,
+    lineno: usize,
+    trim_to_column: Option<usize>,
+    context_lines: Option<usize>,
```

Comment:
I wonder if we really need that. Do we ever configure it anywhere or changed it in the whole history of Sentry? 🤔